mail_inline_css/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Mail Inline CSS",
     "summary": "Convert style tags in inline style in your mails",
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.2",
```

Comment:
Should we not reset to `14.0.1.0.0`?

Reply:
@phuctranerp can you fix it?

Reply:
@phuctranerp @dzungtran89 can you fix it?

Reply:
@simahawk , the PR is updated. Thanks.